src/plugins/intel_cpu/tests/functional/CMakeLists.txt

```diff
@@ -74,43 +73,41 @@ if(NOT (ARM OR AARCH64))
          ${CMAKE_CURRENT_SOURCE_DIR}/shared_tests_instances/low_precision_transformations/aarch64
          ${CMAKE_CURRENT_SOURCE_DIR}/utils/arm)
 else()
-    # temporary disable all custom tests for ARM
+    # temporary disable all custom tests for ARM except explicitly enabled ones
     list(APPEND EXCLUDED_SOURCE_PATHS
         ${CMAKE_CURRENT_SOURCE_DIR}/custom/single_layer_tests
         ${CMAKE_CURRENT_SOURCE_DIR}/custom/subgraph_tests)
-    # except ones which already enabled
-    file(GLOB_RECURSE TMP_LIST_OF_TEST_CLASSES          ${CMAKE_CURRENT_SOURCE_DIR}/custom/single_layer_tests/classes/*.cpp)
```

Comment:
There is no need to remove it.
The Glob pattern is project wide the exclude files can be done by exclude pattern on or different project structure to start search folders on different level.

Is better to keep common approach in the project.

The change will also trigger additional changes in CMake files  on each PR which add new test files which right now is  not required.